src/BlazorUI/Bit.BlazorUI/Utils/Theme/bit-theme.ts

```
@@ -13,6 +14,10 @@ class BitTheme {
             this.currentTheme = options.default;
         }
 
+        if (options.persist) {
+            this.currentTheme = localStorage.getItem(this.THEME_SOTRAGE_KEY) || this.currentTheme;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)